docs/internationalisation.md

```diff
@@ -46,3 +46,17 @@ flutter gen-l10n
 ```
 
 You should see the new strings in the `lib/l10n/lila_*.arb` and `lib/l10n/lila_*.dart` files.
+
+## Adding mobile-specific translations
```

Comment:
👍 

I would add a section on the importance of writing good context information in the `comment` field.

Also another short section explaining that while working on a new feature, hardcoding missing translation in English first, is the preferred way to go.
We want a feature to be done first, and even released and used for a little while. This to avoid useless translating work on non stabilised features.